nfs/src/rpc_task.cpp

```diff
@@ -3132,6 +3146,21 @@ static void readdirplus_callback(
     const bool is_reenumerating =
         (task->rpc_api->readdir_task.get_target_offset() != 0);
 
+#ifdef ENABLE_PRESSURE_POINTS
+            /*
+             * Bad cookie pressure point. Random bad cookie errors which
+             * can cause multiple re-enumerations.
+             */
+            if (inject_error() && status == 0) {
+                AZLogWarn("[{}] PP: bad cookie at offset {} ",
+                          is_reenumerating ? "(R)" : "",
+                          dir_ino, task->rpc_api->readdir_task.get_offset());
+
+                (res)->status = NFS3ERR_BAD_COOKIE;
+                status = task->status(rpc_status, NFS_STATUS(res));
+            }
+#endif
```

Comment:
let's make this a macro like INJECT_JUKEBOX() and use it after INJECT_JUKEBOX(), to not muddle the code